### DISCUSSION TITLE
Welcome to FluentCMS Discussions!

### DISCUSSION THREAD

**Discussion by MostafaTech** [fluentcms/FluentCMS#55] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by AlexandrePViana** [comment]
Hey! Im v3xTV from reddit. Just dropped by to ID myself!

**Reply by pournasserian** [reply]
@AlexandrePViana thanks for joining. I have created a repo for documentation and proposals here:
https://github.com/fluentcms/FluentCMS.Docs

We are writing the MVP's requirements and functionalities there. 

Then, we will extract tasks/issues in the FluentCMS repo.

Feel free to contribute there.

**Comment by ngominhtrungcmag** [comment]
Hi, I thing this is greated CMS for blazor, but how can I access Component for other project blazor? I found Flowbite Tailwind css is so beautiful ui, but this dont have any component for blazor, this project for blazor is pending for 2 year.

**Reply by pournasserian** [reply]
We have developed our own minimal version of components using Flowbite's design (we are not dependent to external component library). See the source here:
https://github.com/fluentcms/FluentCMS/tree/dev/src/Frontend/FluentCMS.Web.UI.Components

You can use any other component library while developing new plugins. Or you can add any CSS for your contents as well.

**Comment by GeorgeZard** [comment]
Hello and good evening awesome project guys i am amazed at the technologies and dynamic ideas you applied i am trying to learn from your awesome examples and build a new template  but since the latest updates when i am trying to build even from a fresh clone of the project during the setup state  i get constant errors first on the IsInitialized check of the setupmanage if i am correct even though the redirect works ok and then during the page load i get constant errors  of the type "Microsoft.JSInterop.JSDisconnectedException: 'JavaScript interop calls cannot be issued at this time. This is because the circuit has disconnected and is being disposed." in SiteBuilder.razor.cs. I am trying to find out what i am doing wrong but i ca't figure it out.
If any more info that i can provide would be useful in recognizing what i am doing wrong i will try to provide it. But i believe it can be easily replicated if you try to build for a clean project.

Thanks in advance and sorry if i am bothering for no reason.
awesome project!

**Reply by pournasserian** [reply]
We are very proud and happy for your great feedback. Many thanks.
Would you please send us a screenshot and/or exception details? 
Feel free to create an issue with details.

**Reply by pournasserian** [reply]
I have seen the exception a few times, which is non-blocking.
Just to be sure, please provide the details. 
Do you need any help on creating a new template?

**Reply by TheHadiAhmadi** [reply]
I've opened an issue for this problem:
https://github.com/fluentcms/FluentCMS/issues/2336

**Reply by GeorgeZard** [reply]
Good day!. Yeah it did not cause any problems it seemed just to be a blazor lifecycle problem but i wasn't sure if i caused it somehow. I will continue with the template so far everything is clear and very beautifully written i am amazed from the process. I will keep in touch with the progress of template if it's ok!  Thanks for the feedback !

**Reply by pournasserian** [reply]
The issue is resolved. Thanks!
